kpatch/kpatch

```diff
@@ -237,6 +237,9 @@ signal_stalled_processes() {
 	if [[ -e "/sys/kernel/livepatch/$module/signal" ]] ; then
 		echo "signaling stalled process(es):"
 		echo 1 > "/sys/kernel/livepatch/$module/signal"
+        else
+            warn "Livepatch process signaling is performed automatically on your system."
+            warn "Skipping manual process signaling."
```

Comment:
I would probably change the term 'manual' by 'explicit' or something, but it doesn't really bother me.

Reply:
I think we've used "manual" in most places when talking about this feature, so my vote would be to keep it as is.

Reply:
I also have a slight preference for "manual". So unless there are more votes for another option I'll leave it as is. :)

Reply:
With @joe-lawrence 's explanation, it makes sense to me. I haven't thought of the other references.